etc/fixtures/add-users.js

```diff
@@ -0,0 +1,72 @@
+// docker exec -it rosalution-rosalution-db-1 mongosh /tmp/fixtures/add-cpam-users.js
+
+const fs = require('fs');
+
+// this file has been removed for security reasons
+// To add users in the future create a csv file in the below path in the following format
+// Name,blazerid,email,clientid,scope
+// John Doe,jdoe,jdoe@site.com,<generated client id>,scope
+
+const inputPath = `/tmp/fixtures/usersToImport.csv`
```

Comment:
Normally I would suggest this to be an input parameter, but I think this script is a good reference for when we create an add user system for how we can incorporate reading csv files.